pkg/segment/segexecution.go

```diff
@@ -112,6 +112,18 @@ func HelperQueryArithmeticAndLogical(queryOps []structs.QueryArithmetic, resMap
 							val = val * -1
 						}
 						finalResult[groupID][timestamp] = val
+					case utils.LetModulo:
+						if swapped {
+							finalResult[groupID][timestamp] = math.Mod(valueRHS, valueLHS)
+						} else {
+							finalResult[groupID][timestamp] = math.Mod(valueLHS, valueRHS)
+						}
```

Comment:
This is fine (this PR can get merged like this)

Sometime later, you might want to refactor this since you have this swapping logic in every `case` of this `switch`. So before this `switch`, you could handle the swapping logic once and then you don't have to do it in each `case`. After that, you could move this whole `switch` to a separate function, since you have the same `switch` (except for the swapping logic) a little lower. That way there's less duplicate code; or you could keep it how it is if you prefer

Reply:
Yeah, right now we only support using one operator in an expression. I think I will modify that when I add support for using multiple operators at a time